src/asciidocParser.ts

```diff
@@ -52,6 +53,7 @@ export class AsciidocParser {
         this.errorCollection.clear()
       }
 
+      this.processor = asciidoctor()
```

Comment:
We should declare `const processor = asciidoctor()` at the top of this file and replace `this.processor` by `processor`.

Reply:
OK, I have taken the long road home...